steelscript-appfwkdev-linux/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "steelscript-appfwkdev-linux",
```

Comment:
Update this to be human readable, eg. "Steelscript Application Framework Dev Box" - 60 chars max